.kokoro/tests/build_cloud_functions.sh

```diff
@@ -56,28 +56,33 @@ fi
 # Deploy functions
 set -x
 
-if [[ "$file" == *"hello-http"* ]]; then
-  echo "Deploying function HelloHttp to: ${FUNCTIONS_HTTP_FN_NAME}"
-  gcloud functions deploy $FUNCTIONS_HTTP_FN_NAME \
-    --region $FUNCTIONS_REGION \
-    --runtime $FUNCTIONS_JAVA_RUNTIME \
-    --entry-point "functions.HelloHttp" \
-    --trigger-http
-elif [[ "$file" == *"hello-pubsub"* ]]; then
-  echo "Deploying function HelloPubSub to: ${FUNCTIONS_PUBSUB_FN_NAME}"
-  gcloud functions deploy $FUNCTIONS_PUBSUB_FN_NAME \
-    --region $FUNCTIONS_REGION \
-    --runtime $FUNCTIONS_JAVA_RUNTIME \
-    --entry-point "functions.${LANGUAGE}HelloPubSub" \
-    --trigger-topic $FUNCTIONS_SYSTEM_TEST_TOPIC
-elif [[ "$file" == *"hello-gcs"* ]]; then
-  echo "Deploying function HelloGcs to: ${FUNCTIONS_GCS_FN_NAME}"
-  gcloud functions deploy $FUNCTIONS_GCS_FN_NAME \
-    --region $FUNCTIONS_REGION \
-    --runtime $FUNCTIONS_JAVA_RUNTIME \
-    --entry-point "functions.HelloGcs" \
-    --trigger-bucket $FUNCTIONS_BUCKET
-fi
+for i in {1..5}; do # Retry up to 5 times
+  if [[ "$file" == *"hello-http"* ]]; then
+    echo "Deploying function HelloHttp to: ${FUNCTIONS_HTTP_FN_NAME}"
```

Comment:
Look at the issue, it looks like the error `An operation on function pubsub-37ee50795f85 in region us-central1 in project java-docs-samples-testing is already in progress. Please try again later.` happens when you try to deploy a new function when the existing on already has an ongoing operation. 

I'm concerned this isn't the right fix because the root cause is a shared function name. It looks like the function name is derived from:
```
SUFFIX=${KOKORO_GITHUB_PULL_REQUEST_NUMBER:-${SAMPLE_VERSION:0:12}}
```

Which is not sufficiently different between parallel builds using the same PR/commit. Can we update the function name to use a uuid instead? Otherwise, it's likely that a retry here will just re-deploy the function and create a race condition between different builds.